pkg/descheduler/strategies/duplicates_test.go

```diff
@@ -313,6 +313,9 @@ func TestFindDuplicatePods(t *testing.T) {
 			sharedInformerFactory.Start(ctx.Done())
 			sharedInformerFactory.WaitForCacheSync(ctx.Done())
 
+			eventBroadcaster, eventRecorder := utils.GetRecorderAndBroadcaster(ctx, fakeClient)
```

Comment:
Given the unit tests do not check the events recorded during testing the `eventRecorder` can be replaced by [NewFakeRecorder](https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/client-go/tools/events/fake.go#L41) which does not need any `eventBroadcaster.Shutdown()`.